### ISSUE TITLE
Update 'add friend' icons

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-Android-XMPP#399]
<img width="681" alt="screenshot 2018-01-19 14 01 02" src="https://user-images.githubusercontent.com/1434597/35169199-3f976a66-fd21-11e7-815c-64a557eea4d5.png">

Please make these the app color (pink) and update the camera icon to https://material.io/icons/#ic_photo_camera

* I'm making this 'high' because it affects the zomkyi script

**Comment by n8fr8** [comment]
The icon you linked to is only available in black or white... not in the style we have above as white with a gray outline.

Do you also mean, the icons themselves should be pink?

**Comment by N-Pex** [comment]
@cstiens Yes, same problem on the iOS side. For now I used the "black" icons tinted pink, but the text label still black. Please advise.

**Comment by N-Pex** [comment]
Related to https://github.com/zom/Zom-iOS/issues/511

**Comment by N-Pex** [comment]
@cstiens Do you have some more info on the colors?
